cesium/featurize.py

```diff
@@ -1,20 +1,37 @@
 import copy
 import os
+from collections import Iterable
 import numpy as np
 import pandas as pd
-from celery import group
-from .celery_tasks import celery_available
-from .celery_tasks import featurize_ts_data as featurize_data_task
-from .celery_tasks import featurize_ts_file as featurize_file_task
+import dask.multiprocessing
+from dask import delayed
+
 from . import data_management
 from . import featurize_tools as ft
 from . import time_series
+from . import util
 from .time_series import TimeSeries
 
 
 __all__ = ['load_and_store_feature_data', 'featurize_time_series']
 
 
+def compatible_shapes(x, y):
```

Comment:
This will necessarily unpack all the iterators.  So, in that case, should we not simply convert to a numpy array before proceeding?